scripts/verdaccio.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+ORIGARGS=("$@")
+
+helpFunction() {
+	echo "Usage: $0 [start|stop|publish|startAndPublish] [background]"
+	exit 1 # Exit script after printing help
+}
+
+start() {
+	. scripts/env.sh
+	if [[ ${ORIGARGS[1]} == "background" ]]; then
+        startBackground
+	else
+		echo "Starting verdaccio..."
+        docker run -it --rm --name verdaccio -p 4873:4873 verdaccio/verdaccio
+	fi
+}
+
+startBackground() {
+    echo "Starting verdaccio in background..."
+    docker run -d --rm --name verdaccio -p 4873:4873 verdaccio/verdaccio
+}
+
+stop() {
+	echo "Stopping verdaccio ..."
+    docker ps -q --filter ancestor="verdaccio/verdaccio" | xargs -r docker stop
+}
+
+createVerdaccioNPMUser() {
+    curl -XPUT \
+        -H "Content-type: application/json" \
+        -d '{ "name": "test", "password": "test" }' \
+        'http://localhost:4873/-/user/org.couchdb.user:test'
+}
+
+loginNPMUser() {
+    npm-auth-to-token \
+        -u test \
+        -p test \
+        -e test@test.com \
+        -r http://localhost:4873
+}
+
+lernaUpdatePackageVersions() {
+    lerna version 5.0.0 \
+        --ignore-scripts \
+        --no-push \
+        --no-private \
+        --no-git-tag-version \
+        --yes
+}
+
+lernaBuildAndCommit() {
+    yarn build
+
+    git add .
+    git commit -m "Comitting for black box publish"
+}
+
+lernaPublish() {
+    lerna publish from-package \
+        --no-git-tag-version \
+        --no-push \
+        --registry http://localhost:4873 \
```

Comment:
We are using local registry here but an extra step to avoid risk , 
add some tag other then default `latest` , may be `alpha` or `rc`